ludwig/decoders/llm_decoders.py

```diff
@@ -110,23 +110,31 @@ def get_prediction_set(self):
 
     def forward(self, inputs: List[torch.Tensor], **kwargs):
         # Extract the sequences tensor from the LLMs forward pass
+        max_new_tokens = kwargs.get("max_new_tokens", self.max_new_tokens)
```

Comment:
Conditionally grabbing `max_new_tokens` from either state or kwargs feels brittle. Can we make `max_new_tokens` a parameter that's always passed in and always use that?

Reply:
done